website/src/pages/api/update_task.ts

```diff
@@ -36,6 +36,7 @@ const handler = async (req, res) => {
 
   // Send the interaction to the Task Backend.  This automatically fetches the
   // next task in the sequence (or the done task).
+  // TODO Move this into OasstApiClient.
   const interactionRes = await fetch(`${process.env.FASTAPI_URL}/api/v1/tasks/interaction`, {
```

Comment:
Are you tackling this in a follow up PR?

Reply:
bdcfae5